http/auth_test.go

```diff
@@ -24,7 +24,7 @@ import (
 // NewMockAuthorizationBackend returns a AuthorizationBackend with mock services.
 func NewMockAuthorizationBackend() *AuthorizationBackend {
 	return &AuthorizationBackend{
-		Logger: zap.NewNop().With(zap.String("handler", "authorization")),
+		log: zap.NewNop().With(zap.String("handler", "authorization")),
```

Comment:
This seems redundant. It's a no-op logger, so looks like the `With` can be removed?